cvat-canvas/src/typescript/masksHandler.ts

```diff
@@ -294,6 +295,42 @@ export class MasksHandlerImpl implements MasksHandler {
         }
     }
 
+    private checkEmpty(): void {
+        if (this.drawnObjects.length === 0) {
+            this.tool.configureBlockedTools({
+                eraser: true,
+                'polygon-minus': true,
+            });
+            return;
+        }
+        const wrappingBbox = this.getDrawnObjectsWrappingBox();
+        if (this.brushMarker) {
+            this.canvas.remove(this.brushMarker);
+        }
+        const imageData = this.imageDataFromCanvas(wrappingBbox);
+        if (this.brushMarker) {
+            this.canvas.add(this.brushMarker);
+        }
+        const rle = zipChannels(imageData);
+        const emptyMask = rle.length < 2;
+        this.tool.configureBlockedTools({
+            eraser: emptyMask,
+            'polygon-minus': emptyMask,
+        });
```

Comment:
1. Let's wrap adding brushMarker back to try ... finally 
2. Probably we can rewrite as `private funcName = lodash.debounce( ... )`
3. I would rename as `updateBlockedTools` or something like that

Reply:
Renamed.
Why do we need to wrap it in try catch? I doesnt look we use try anywhere with canvas.add?
I dont think we need to rewrite with `private funcName = lodash.debounce( ... )` because this funtion is used in proxy wrapper where we cant really access `this` without alias